modules/account-lib/src/coin/eth/utils.ts

```diff
@@ -479,10 +486,10 @@ export function getBufferedByteCode(methodId: string, rawData: string): Buffer {
  * @param tokenContractAddress The contract address to match against
  * @returns statics BaseCoin object for the matching token
  */
-export function getToken(tokenContractAddress: string): Readonly<BaseCoin> | undefined {
+export function getToken(tokenContractAddress: string, network : BaseNetwork): Readonly<BaseCoin> | undefined {
   const tokens = coins.filter((coin) => {
```

Comment:
All expression can be simplified:

```
  const tokens = coins.filter((coin) => coin instanceof ContractAddressDefinedToken) &&
      coin.network === network && 
      coin.contractAddress.toLowerCase() === tokenContractAddress.toLowerCase())
```

Reply:
thank you, I think it's ok to keep it for readability